controls/tests.py

```diff
@@ -821,16 +813,16 @@ def test_smt_autocomplete(self):
         search_comps_txtbar[-1].click()
         search_comps_txtbar[-1].clear()
         search_comps_txtbar[-1].send_keys("Component")
-        self.browser.find_elements_by_id("selected_producer_element_form_id")[-1].click()
-        var_sleep(3)
+        self.browser.find_elements_by_id("selected_producer_element_form_id")[-1].click()# Ajax request
+        var_sleep(2)
```

Comment:
multiple var_sleep calls here can probably be swapped out, but I'm curious if that's not the case.

Reply:
yeah ajax like file uploads/downloads are edge cases that force the need for var_sleeps